BoxContentSDK/BoxContentSDK/Requests/BOXRequest.m

```diff
@@ -325,8 +325,7 @@ - (NSString *)fullFolderFieldsParameterString
                        BOXAPIObjectKeyExtension,
                        BOXAPIObjectKeyIsPackage,
                        BOXAPIObjectKeyAllowedSharedLinkAccessLevels,
-                       BOXAPIObjectKeyCollections,
-                       BOXAPIObjectKeyRepresentations];
+                       BOXAPIObjectKeyCollections];
```

Comment:
Per discussion we are moving representation to support high definition video file request only.